### ISSUE TITLE
Android Messages - Refreshing Service Causes Google 400 Error "The requested URL was not found on this server. That’s all we know."

### ISSUE THREAD

**Issue by zkhcohen** [meetfranz/plugins#417]
Android Messages frequently struggles to connect with the mobile app. This naturally prompts the user to refresh the service within Franz.

Upon refreshing the service, it displays the Google 400 page - "The requested URL was not found on this server. That’s all we know." Using the Google Debugging Tools, I see that it 302s, then 400s on the GET for "https://messages.google.com/web".

Further refreshes and disabling/re-enabling the service don't appear to resolve the issue. The only solutions are to completely remove and re-add the service, or to unregister the Service Workers through Developer Tools > Application > Clear Application Storage > Unregister Service Workers. Both solutions will force you back to the authentication page.

If I actually go to the Application > Service Workers page in Developer Tools and Unregister the latest "serviceworker-bundle.js" manually, then reload the Messages Franz Service, it will refresh and successfully put you back into the Messages Service without pushing you through the Authentication process again.

{"host":{"platform":"win32","release":"10.0.18362"....
{"id":"android-messages","version":"1.2.0"}

### Service
Android Messages

### Link to your repository
https://github.com/dweinber/franz-recipe-android-messages 

### Additional Information
N/A

**Comment by zkhcohen** [comment]
My mistake, it looks like other people discovered this issue and workaround as well. I see the issue referenced here:

https://github.com/meetfranz/plugins/issues/334
https://github.com/meetfranz/franz/issues/1583
https://github.com/meetfranz/franz/issues/1575
https://github.com/meetfranz/franz/issues/1588

**Comment by stacksjb** [comment]
Yup, that's been my go to is clicking "unregister" and then reloading the service. Maybe someone could patch the plugin to unregister it on reload
